### PR TITLE
API for get a 'Checkout'

### LINKED ISSUE #31: Missing API: GET: /project/:username/:project/checkout-key
It seems to be "API to Lists checkout keys" is not implemented by this gem.

### PR DESCRIPTION
**Pull request by anujaware**
Fixes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ end
   * [Recent Builds](#recent_builds)
   * [Settings](#settings)
   * [Unfollow](#unfollow)
+  * [Checkout](#checkout)
```
Comment (mtchavez) [inline README.md]: Endpoints are in alphabetical order, please fix.

File: README.md [hunk 2]
```diff
@@ -453,6 +454,31 @@ Example response
     "followed": false
 }
 ```
+#### [checkout](#checkout)
```
Comment (mtchavez) [inline README.md]: Please order this section alphabetically with the other sections.

File: lib/circleci/project.rb [hunk 3]
```diff
@@ -130,7 +130,22 @@ def self.settings username, project
...
 
+
+    ##
+    #
+    # Gets the checkout key
+    #
+    # @param username     [String] - User or org name who owns project
+    # @param project      [String] - Name of project
+    # @param fingerprint  [String] - Fingerprint
+    # @return         [CircleCi::Response] - Response object
```
Comment (mtchavez) [inline lib/circleci/project.rb]: Indentation should match the other parameter types please.

**Comment by mtchavez** [comment]
Please rebase against master and make the changes addressed in the comments. This looks good otherwise, thanks.

**Comment by mtchavez** [comment]
Please add pull request descriptions in the future.

**Comment by anujaware** [comment]
Done!

**Pull request closed without merging**